schemaregistry-avro/src/main/scala/com/microsoft/azure/schemaregistry/spark/avro/SchemaRegistryExceptions.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.azure.schemaregistry.spark.avro
+
+final case class MissingPropertyException(private val message: String) extends Exception(message) {}
+
+final case class SchemaNotFoundException(private val message: String) extends Exception(message) {}
```

Comment:
I might be missing it but I don't seem to see where we use this?

Reply:
you are right that SchemaNotFoundException is not in use